src/steps/relay.py

```diff
@@ -128,3 +159,18 @@ def create_message(self, **kwargs):
     def subscribe_and_publish_with_retry(self, node_list, pubsub_topic_list):
         self.ensure_relay_subscriptions_on_nodes(node_list, pubsub_topic_list)
         self.check_published_message_reaches_relay_peer()
+
+    @allure.step
+    def register_rln_single_node(self, **kwargs):
+        step_id = f"{datetime.now().strftime('%Y-%m-%d_%H-%M-%S')}__{str(uuid4())}"
+        logger.debug(f"Registering RLN credentials for single node: {inspect.currentframe().f_code.co_name}")
+        self.node1 = WakuNode(DEFAULT_NWAKU, f"node1_{step_id}")
+        self.node1.register_rln(rln_creds_source=kwargs["rln_creds_source"], rln_creds_id=kwargs["rln_creds_id"])
+
+    @allure.step
+    def check_rln_registration(self, key_id):
+        creds_file_path = "/keystore_" + key_id + "/keystore.json"
+        if rln_credential_store_ready(creds_file_path):
+            return True
+        else:
```

Comment:
I think you need a try except block if you want to return False. Because if rln_credential_store_ready will return Excetption if not True

Reply:
Let's follow the Pythonic way https://github.com/waku-org/waku-interop-tests/pull/16/commits/1c901c7c7f7d1e92408f988a76c6ba43f8199ef1